### PR TITLE
os/board/rtl8721csm/hal: enable the loopback mode in UART

### PR DESCRIPTION
**Pull request by deepaksrma**

### CODE REVIEW

File: apps/examples/testcase/ta_tc/systemio/itc/itc_uart.c [hunk 1]
```diff
@@ -360,8 +360,8 @@ static void itc_systemio_iotbus_uart_write_read_flush_p(void)
 static void itc_systemio_iotbus_uart_write_read_flush_p_multi_handle(void)
 {
 	int ret = IOTBUS_ERROR_NONE;
-	char sz_input_text1[BUF_LEN] = "UART READ/WRITE ITC TESTING!";
-	char sz_input_text2[BUF_LEN] = "UART READ/WRITE ITC TESTING1!";
+	char sz_input_text1[BUF_LEN] = "UART READ/WRITE";
+	char sz_input_text2[BUF_LEN] = "UART READ/WRITE1";
```
Comment (sangam-swami) [inline apps/examples/testcase/ta_tc/systemio/itc/itc_uart.c]: make it uniform WRITE2

File: apps/examples/testcase/ta_tc/systemio/itc/itc_uart.c [hunk 2]
```diff
@@ -636,7 +638,7 @@ static void itc_systemio_iotbus_uart_set_mode_flush_write_read_p(void)
 	int i_bytesize = 8;
 	int i_stop_bits = 1;
 	int ret = IOTBUS_ERROR_NONE;
-	char sz_input_text[BUF_LEN] = "UART READ/WRITE ITC TESTING!";
+	char sz_input_text[BUF_LEN] = "UART READ/WRITE";
```
Comment (sangam-swami) [inline apps/examples/testcase/ta_tc/systemio/itc/itc_uart.c]: UART READ/WRITE TESTING

File: lib/libc/termios/lib_loopback_en.c [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+ ****************************************************************************/
+
+/****************************************************************************
+ * Name: loopback_en
+ *
+ * Description:
+ *   calls the ioctl to enable/ disable the loopback mode of UART
+ *
+ * Input Parameters:
+ *   fd  - The 'fd' argument is an open file descriptor associated with a terminal.
```
Comment (sangam-swami) [inline lib/libc/termios/lib_loopback_en.c]: struct termios also you can add in input list

File: lib/libc/termios/lib_loopback_en.c [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+
+#include <sys/ioctl.h>
+
+#include <termios.h>
+#include <errno.h>
+
```
Comment (sangam-swami) [inline lib/libc/termios/lib_loopback_en.c]: all these headers needed?

File: apps/examples/testcase/ta_tc/systemio/itc/itc_uart.c (hunk 1 above)
Comment (deepaksrma) [reply apps/examples/testcase/ta_tc/systemio/itc/itc_uart.c]: corrected

File: apps/examples/testcase/ta_tc/systemio/itc/itc_uart.c (hunk 2 above)
Comment (deepaksrma) [reply apps/examples/testcase/ta_tc/systemio/itc/itc_uart.c]: corrected

File: lib/libc/termios/lib_loopback_en.c (hunk 3 above)
Comment (deepaksrma) [reply lib/libc/termios/lib_loopback_en.c]: corrected

File: lib/libc/termios/lib_loopback_en.c (hunk 4 above)
Comment (deepaksrma) [reply lib/libc/termios/lib_loopback_en.c]: corrected

**Pull request closed without merging**